transforms/code/repo_level_ordering/ray/src/dpk_repo_level_order/internal/repo_grouper.py

```diff
@@ -0,0 +1,151 @@
+import logging
+import os
+from typing import List, Tuple
+
+import pandas as pd
+import pyarrow as pa
+import ray
+from data_processing.utils import get_logger
+from data_processing_ray.runtime.ray import RayUtils
+from pyarrow.fs import FileSelector, FileType, LocalFileSystem, S3FileSystem
+from pyarrow.parquet import ParquetDataset, write_table
+
+
+class DataAccessAlternative:
+    def __init__(self, key=None, secret=None, endpoint=None):
+        if key and secret:
+            self.fs = S3FileSystem(
+                access_key=key,
+                secret_key=secret,
+                endpoint_override=endpoint,
+                request_timeout=20,
```

Comment:
request_timeout and connect_time is hardcoded. can we parameterized with default.

Reply:
done